testsuite/integration/basic/pom.xml

```diff
@@ -235,6 +235,7 @@
                                         <exclude>org/jboss/as/test/integration/ee/injection/resource/basic/*TestCase.java</exclude>
                                         <exclude>org/jboss/as/test/integration/ee/naming/defaultbindings/**/*TestCase.java</exclude>
                                         <exclude>org/jboss/as/test/integration/jca/moduledeployment/**/*TestCase*.java</exclude>
+                                        <exclude>org/jboss/as/test/integration/messaging/mgmt/JMSBridgeManagementTestCase.java</exclude>
```

Comment:
org/jboss/as/test/integration/messaging/**/*TestCase*.java is already excluded, this should not be nessesary